stitching_base/ffmpeg.sh.arm64

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+sudo apt update -y
+sudo apt remove -y ffmpeg
+sudo apt autoremove -y
+sudo apt remove --purge -y ffmpeg
+
+mkdir ffmpeg_top
+cd ffmpeg_top
+
+git clone https://github.com/bepro-company/jetson-ffmpeg.git
+cd jetson-ffmpeg
+mkdir build
+cd build
+cmake ..
+make -j$(nproc)
+sudo make install
+sudo ldconfig
+
+cd ../..
+git clone git://source.ffmpeg.org/ffmpeg.git -b release/4.2 --depth=1
+cd ffmpeg
+wget https://github.com/jocover/jetson-ffmpeg/raw/master/ffmpeg_nvmpi.patch
```

Comment:
Why is this from original repo but upper one from bepro? I think so better to use from bepro repo.